src/scss/constants.scss

```diff
@@ -1,11 +1,19 @@
 $primaryColor: var(--primary-color);
 $backgroundColor: var(--background-color);
 
+$primary--lightest: #a179ff;
+$primary--light: #9669ff;
 $primary: #6f43ff;
-$dimmer-primary: #3f247d;
+$dimmer--darker: #3f247d;
 
+$secondary--lightest: #868686;
+$secondary--light: #6a6a6a;
+$secondary: #333035;
+$secondary--dark: #19181a;
+$secondary--darkest: #090909;
```

Comment:
Note: It's my subjective opinion. Would be good to refine it with @mcflyok, but it seems to reflect the current state of the app in a decent manner